app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, dependent: :destroy
```

Comment:
**You did a great job! I especially like the `seeds.rb`**

- [ ] However, I can see that you forgot to apply the changes given by the previous reviewer. As mentioned above, could you add foreign_key to your comments and likes?

Reply:
@Rustamxon7,  you may miss the conversation above with the previous reviewer. He asked me to add it because he thought foreign key name should be different like `author_id`.
![image](https://user-images.githubusercontent.com/55840999/228671846-c4971f59-c73f-4f51-bcc6-8989ffa67ff0.png)

But as per the [documentation](https://guides.rubyonrails.org/association_basics.html), it's not necessary to add foreign key options to the has_many relation for comments and likes in this code since the foreign key columns are named in accordance with Rails' naming convention. By default, Rails will assume that the foreign key for a `has_many` association is the lowercase singular name of the association followed by `_id`. In this case, the foreign key for `posts` association is `author_id`, and for `comments` and `likes` associations it's `user_id`, which matches the naming convention.

Reply:
Hi @shahadat3669, 
Sorry for the late reply!

I wrote you via slack!